util/travis/before_install.sh

```diff
@@ -9,7 +9,35 @@ if [ $CC = "clang" ]; then
 	sudo apt-get install clang
 fi
 sudo apt-get install p7zip-full
-if [ $WINDOWS = "no" ]; then
+if [ $ANDROID = "yes" ]; then
+	
+	printf "Generating swap..."
```

Comment:
It has been in order to test whether RAM is an issue. I do some experimental changes, as I don't have a local travis server. Btw it doesn't even work, as travis forbids swap.
The error that makes this unreliable is `internal compiler error: Killed (program cc1)`. If you google for it, most results point into the RAM's direction. If you have a better idea, please suggest a way to resolve it :).